### PR TITLE
AT_03.007 | Footer | Link REST API Redirect

### PR DESCRIPTION
**Pull request by ElenaKanash**
https://trello.com/c/qAA9XSxq

### CODE REVIEW

File: cypress/e2e/footerRestApiLink.cy.js [hunk 1]
```diff
@@ -10,4 +11,10 @@ describe('footerRestApiLink', () => {
         cy.get('.page-footer__links a[href="api/').click();
         cy.get('#main-panel h1').should('have.text', 'REST API');
     })
+
+    it('AT_03.007 | Footer | Link REST API Redirect', () => {
+        cy.get('.page-footer a[href="api/"]').invoke('removeAttr', 'target').click()
+        cy.url().should('equal', `http://localhost:${LOCAL_PORT}/api/`)
+        cy.title().should('equal', 'Remote API [Jenkins]')
```
Comment (OLgaByR) [inline cypress/e2e/footerRestApiLink.cy.js:18]: general recommendation - assert (as well as  precondition) should be visually separated from test body with empty line added ( between 17 and 18 lines)

**Review by OLgaByR — APPROVED** [review]

**Review by 7108581 — APPROVED** [review]

File: cypress/e2e/footerRestApiLink.cy.js (hunk 1 above)
Comment (ElenaKanash) [reply cypress/e2e/footerRestApiLink.cy.js:19]: Thank you for your comment. Added.

File: cypress/e2e/footerRestApiLink.cy.js [hunk 2]
```diff
@@ -10,4 +11,11 @@ describe('footerRestApiLink', () => {
         cy.get('.page-footer__links a[href="api/').click();
         cy.get('#main-panel h1').should('have.text', 'REST API');
     })
+
+    it('AT_03.007 | Footer | Link REST API Redirect', () => {
+        cy.get('.page-footer a[href="api/"]').invoke('removeAttr', 'target').click()
+        
+        cy.url().should('equal', `http://localhost:${LOCAL_PORT}/api/`)
+        cy.title().should('equal', 'Remote API [Jenkins]')
```
Comment (DariaMas7) [inline cypress/e2e/footerRestApiLink.cy.js:19]: Remote API [Jenkins] its better to move it to fixtures please

File: cypress/e2e/footerRestApiLink.cy.js (hunk 1 above)
Comment (OLgaByR) [reply cypress/e2e/footerRestApiLink.cy.js:19]: I approved Pr as it was, you can refactor later, but now you added lines between two asserts you have - it is unnecessary to separate  asserts, line 19 is excessive

**Pull request merged**